modules/social_features/social_user/social_user.install

```diff
@@ -208,3 +208,45 @@ function social_user_update_11402() : void {
   ]);
   $form_mode->save();
 }
+
+/**
+ * Update 'views.view.user_admin_people'.
+ *
+ * Replace the login filter to last access and change label of access field.
+ */
+function social_user_update_11403(array &$sandbox): void {
```

Comment:
@Kovalskiy266 since we agreed to move it to milestone 12, please change update hook number to `12001`

Reply:
Done